src/components/Quote.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect } from 'react';
+import './css/quote.css';
```

Comment:
- [ ] You've done an excellent job with the Quote component, but kindly review the imports in your project. I'm encountering an error when attempting to run your application, and it appears that the quote.css file is missing from your project.

![Screenshot from 2023-08-26 10-08-11](https://github.com/misikir21/mama/assets/105117832/bb599403-2436-47eb-86fb-7529e674642a)